Tasks/DockerComposeV0/utils.ts

```diff
@@ -9,7 +9,7 @@ export function getFinalComposeFileName(): string {
     return ".docker-compose." + Date.now() + ".yml"
 }
 
-export function writeFileSync(filename: string, data: any, options?: { encoding?: string; mode?: number; flag?: string; }): void {
+export function writeFileSync(filename: string, data: any, options?: { encoding?: BufferEncoding; mode?: number; flag?: string; }): void {
```

Comment:
should this also be #if NODE16 ?  Or is there little of changing it?

Reply:
Thanks for the notice! I wrapped it in `#if NODE16`